server/src/domain/search/search.service.ts

```diff
@@ -122,4 +130,45 @@ export class SearchService {
     userIds.push(...partnersIds);
     return userIds;
   }
+
+  async getSearchSuggestions(auth: AuthDto, dto: SearchSuggestionRequestDto): Promise<SearchSuggestionResponseDto> {
+    let response: SearchSuggestionResponseDto = { data: [] };
+
+    if (dto.type === SearchSuggestionType.Country) {
+      const country = await this.metadataRepository.getCountries(auth.user.id);
+      response = {
+        data: country,
+      };
```

Comment:
```suggestion
      return this.metadataRepository.getCountries(auth.user.id);
```

Especially if you make the return type for this api just `string[]` you can just do this.